bindings/python/tests/test_collection.py

```diff
@@ -114,6 +114,12 @@
 i1, = coll.get_instances()
 assert i1 == inst1
 
+# We have no collections in the collection
```

Comment:
why do we do this test? why would it be a problem?

Reply:
It is not a problem, we just test that get_instances() behave as expected.

On line 118 we test get_instances() with a metaid referring to a collection. Since we have no nested collection in the collection, the returned iterator should be empty.